src/foam/mlang/mlang.js

```diff
@@ -1695,7 +1695,7 @@ foam.CLASS({
       return typeof x === 'number' ? '' + x :
         typeof x === 'string' ? '"' + x + '"' :
         Array.isArray(x) ? '[' + x.map(this.toString_.bind(this)).join(', ') + ']' :
-        x.toString ? x.toString() :
+        (x && x.toString) ? x.toString() :
```

Comment:
Don't all JS objects have toString(), since it's in the Object class?

Reply:
x here is the value of foam.mlang.Constant and it can be null then throws on x.toString.